### PR TITLE
Update grammar and content

### PR DESCRIPTION
**Pull request by bryantson**
Reviewed grammar and made some changes:
https://docs.google.com/document/d/1YqZxtpy37-vk5DnnzE3t5clLAaA6PS6F_pyN7FHGP4M/edit?tab=t.0

Documentation updates:

* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L5-R37): Enhanced the introduction and objectives section, added a note about the workshop being conducted through Codespace, and included estimated times for each section in the Table of Contents.
* [`docs/1_Story/README.md`](diffhunk://#diff-01d43080054adcc745983790aff383e4408c8fca0ffe5b41cefe416a97ebb884L7-R22): Improved formatting by bolding key terms, added a note about the limited workshop time, and corrected a broken link. [[1]](diffhunk://#diff-01d43080054adcc745983790aff383e4408c8fca0ffe5b41cefe416a97ebb884L7-R22) [[2]](diffhunk://#diff-01d43080054adcc745983790aff383e4408c8fca0ffe5b41cefe416a97ebb884L43-R45) [[3]](diffhunk://#diff-01d43080054adcc745983790aff383e4408c8fca0ffe5b41cefe416a97ebb884L64-R91) [[4]](diffhunk://#diff-01d43080054adcc745983790aff383e4408c8fca0ffe5b41cefe416a97ebb884L98-R100)

New resources:

* [`docs/7_Troubleshooting/README.md`](diffhunk://#diff-09c5d63cbf75384a434748bf1cf20a71b5989c523edc1fa10eb30753ac50e319R1-R11): Added a new troubleshooting guide and FAQ section to assist participants with common issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,32 +2,36 @@
...
-## Table of Contents
+## Table of Contents (Total Estimated Time: About 30 minutes)
 
-1. [The OctoFit Tracker app story](docs/1_Story/README.md)
-2. [Prerequisites and asetup of the development environment](docs/2_Prerequisites/README.md)
-3. [Getting started - app frontend and backend creation](docs/3_GettingStarted/README.md)
-4. [The OctoFit Tracker database and app backend creation](docs/4_BackendSettings/README.md)
-5. [Populate the database with sample data](docs/5_PopulateDBwData/README.md)
-6. [Using the Codespace endpoint to access the Django REST Framework](docs/6_CodespaceDjangoRESTFramework/README.md)
+1. [The OctoFit Tracker app story](docs/1_Story/README.md) - :alarm_clock: Estimated Time: 2 minutes
```
Comment (arilivigni) [inline README.md:31]: I rather not put times on these. I don't think that is necessary

Comment (bryantson) [reply README.md:31]: Estimated time?

**Review by arilivigni — DISMISSED** [review]
Just remove times otherwise looks good

File: README.md (hunk 1 above)
Comment (bryantson) [reply README.md]: Removed

**Pull request merged**